packages/react-components/react-utilities/src/compose/getIntrinsicElementProps.ts

```diff
@@ -19,7 +19,7 @@ export const getIntrinsicElementProps = <
   /** The slot's default element type (e.g. 'div') */
   tagName: NonNullable<Props['as']>,
   /** The component's props object */
-  props: Props & React.RefAttributes<InferredElementRefType<Props>>,
```

Comment:
this is missing in https://github.com/microsoft/fluentui/pull/34456, wondering if that's on purpose or oversight ?

Reply:
based on offline convo - this change was reverted with added reasoning to eslint-disable pragma